app/concerns/fetchable.rb

```diff
@@ -3,7 +3,38 @@ module Fetchable
 
   OK_STATUS = "OK"
   UNSUPPORTED_EXTENSION = "URL extension is not one we index"
-  BLACKLISTED_EXTENSIONS = %w{wmv mov css csv gif htc ico jpeg jpg js json mp3 png rss swf txt wsdl xml zip gz z bz2 tgz jar tar m4v}
```

Comment:
The `bz2` omission is an oops (which is weird, because I double-checked the list - further evidence that I'm losing it). I thought `z` was a typo, but little do I know. (At some point I'd like to extend this list more thoroughly, but for the time being I'm just adding a few that I've encountered in crawls.)